### PR TITLE
[11.0][ADD] stock_inventory_analytic

### PR DESCRIPTION
**Pull request by HviorForgeFlow**
This module allow to add analytic accounts on stock inventory line whether using Inventory Adjustments or updating quantity on hand product wizard.

### CODE REVIEW

File: stock_inventory_analytic/views/stock_inventory_line_view.xml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+
+    <record id="stock_analytic_inventory_line_tree" model="ir.ui.view">
+        <field name="name">stock.inventory.line.tree</field>
+        <field name="model">stock.inventory.line</field>
+        <field name="inherit_id" ref="stock.stock_inventory_line_tree"/>
+        <field name="arch" type="xml">
+            <field name="location_id" position="after">
+                <field name="analytic_account_id"/>
```
Comment (JordiBForgeFlow) [inline stock_inventory_analytic/views/stock_inventory_line_view.xml]: add `groups="analytic.group_analytic_accounting"`

File: stock_inventory_analytic/views/stock_inventory_line_view.xml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        </field>
+    </record>
+
+    <record id="stock_analytic_inventory_line_tree2" model="ir.ui.view">
+        <field name="name">stock.inventory.line.tree</field>
+        <field name="model">stock.inventory.line</field>
+        <field name="inherit_id" ref="stock.stock_inventory_line_tree2"/>
+        <field name="arch" type="xml">
+            <field name="location_id" position="after">
+                <field name="analytic_account_id"/>
```
Comment (JordiBForgeFlow) [inline stock_inventory_analytic/views/stock_inventory_line_view.xml]: `groups="analytic.group_analytic_accounting"`

File: stock_inventory_analytic/views/stock_inventory_line_view.xml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        </field>
+    </record>
+
+    <record id="view_stock_analytic_inventory_form" model="ir.ui.view">
+        <field name="name">stock.inventory.form.inherit</field>
+        <field name="model">stock.inventory</field>
+        <field name="inherit_id" ref="stock.view_inventory_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='line_ids']/tree/field[@name='location_id']" position="after">
+                <field name="analytic_account_id"/>
```
Comment (JordiBForgeFlow) [inline stock_inventory_analytic/views/stock_inventory_line_view.xml]: `groups="analytic.group_analytic_accounting"`

File: stock_inventory_analytic/views/stock_inventory_line_view.xml [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+            </xpath>
+            <xpath expr="//field[@name='line_ids']/kanban/field[@name='location_id']" position="after">
+                <field name="analytic_account_id"/>
```
Comment (JordiBForgeFlow) [inline stock_inventory_analytic/views/stock_inventory_line_view.xml]: `groups="analytic.group_analytic_accounting"`

File: stock_inventory_analytic/wizard/stock_product_change_qty.xml [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+
+    <record id="view_change_product_quantity_analytic" model="ir.ui.view">
+        <field name="name">stock.change.product.qty.analytic</field>
+        <field name="model">stock.change.product.qty</field>
+        <field name="inherit_id" ref="stock.view_change_product_quantity"/>
+        <field name="arch" type="xml">
+            <field name="new_quantity" position="after">
+                <field name="analytic_account_id"/>
```
Comment (JordiBForgeFlow) [inline stock_inventory_analytic/wizard/stock_product_change_qty.xml]: `groups="analytic.group_analytic_accounting"`

**Review by JordiBForgeFlow — APPROVED** [review]
:+1: small comments, but overall looks good to me!

**Comment by HviorForgeFlow** [comment]
Thanks for reviewing, comments attended

File: stock_inventory_analytic/tests/test_stock_inventory_analytic.py [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+            values.update({'analytic_account_id': analytic_account_id.id})
+        wizard = self.wizard_model.create(values)
+        wizard.change_product_qty()
+
+    def test_product_change_qty_analytic(self):
+        product = self._create_product('product_product')
+
+        self._product_change_qty(product, 10, self.analytic_account)
+
+        analytic_line = self.env['account.analytic.line'].search(
```
Comment (rousseldenis) [inline stock_inventory_analytic/tests/test_stock_inventory_analytic.py]: @hveficent IMHO, you have to make a search before calling product_change_qty, take the recordset before, then call a second search and make 'recordset_before - recordset_after' to ensure the analytic line is well created.

Comment (HviorForgeFlow) [reply stock_inventory_analytic/tests/test_stock_inventory_analytic.py]: It makes more sense

**Review by rousseldenis — APPROVED** [review]
Code review

**Review by bjeficent — APPROVED** [review]
Functional test. LGTM

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: Functional review

**Pull request merged**